inst/ProjectTemplate/Pkg/README.md

```diff
@@ -1,9 +1,4 @@
----
-title: "Instructions"
-author: "{{AUTHOR_NAME}}"
-date: "8/10/2020"
-output: html_document
----
+
```

Comment:
Why this section is removed?

Reply:
I converted from RMD to standard markdown here, so you don't generally use the header in .md files I've seen.